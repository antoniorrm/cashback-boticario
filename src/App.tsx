import React from 'react'
import { ThemeProvider } from '@material-ui/core'
import './App.css'
import theme from './theme'
import Routes from './Routes'

function App() {
	return (
		<ThemeProvider theme={theme}>
			<Routes />
		</ThemeProvider>
	)
}

export default App
