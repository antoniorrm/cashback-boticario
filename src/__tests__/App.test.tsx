/* eslint-disable no-undef */
import React from 'react'
import { render } from '@testing-library/react'
import App from '../App'

describe('App component', () => {
	test('renders App component correctly', () => {
		const { container } = render(<App />)
		expect(container).toMatchSnapshot()
	})
})
