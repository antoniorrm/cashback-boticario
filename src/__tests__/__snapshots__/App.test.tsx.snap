// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App component correctly 1`] = `<div />`;
